### ISSUE TITLE
Pages restart

### ISSUE THREAD

**Issue by Gangsta0607** [minbrowser/min#1057]
Hello. Please add 'restart page' button. I don't know, maybe I didn't noticed it, but still.

**Comment by shalva97** [comment]
Did you mean reload page? there is only keyboard shortcut for that CTRL + R

**Comment by Syndamia** [comment]
I think a dedicated button, like in `Firefox`, `Chrome`, ... might clutter a bit the UI. 

However, an option in the top left menu (*three dots*) might be a nice addition (*maybe under the `File` option?*). Perhaps even an option in the (right-click) context menu.

**Comment by PalmerAL** [comment]
I agree an option in the context menu would work well; this is also what someone suggested on the survey a few days ago (I think):

> more popup menu items, such as "reload page", "print page", etc
